lib/osml/data_intake/di_dataplane.ts

```diff
@@ -25,21 +25,23 @@ export class DIDataplaneConfig {
    * @param {string} LAMBDA_ROLE_NAME - The name of the DI Lambda execution role.
    * @param {string} LAMBDA_FUNCTION_NAME - The name of the Lambda for the Data Intake.
    * @param {number} LAMBDA_MEMORY_SIZE - The memory in Mb to give the lambda runtime.
+   * @param {number} LAMBDA_RETRY_ATTEMPTS - The number of times to retry a failed function call.
    * @param {number} LAMBDA_STORAGE_SIZE - The size of the storage to assign lambda runtime in GB.
    * @param {number} LAMBDA_TIMEOUT - The timeout, in seconds, for the Lambda function.
    * @param {string} SNS_INPUT_TOPIC_NAME - The name to give the input SNS topic.
-   * @param {string} SNS_OUTPUT_TOPIC_NAME - The name to give the output SNS topic.
+   * @param {string} SNS_STAC_TOPIC - The name to give the output SNS topic.
    * @param {string} S3_INPUT_BUCKET_NAME - The name to give the input bucket.
    * @param {string} S3_OUTPUT_BUCKET_NAME - The name to give the output bucket.
    */
   constructor(
     public LAMBDA_ROLE_NAME: string = "DILambdaRole",
     public LAMBDA_FUNCTION_NAME: string = "DILambda",
     public LAMBDA_MEMORY_SIZE: number = 4096,
+    public LAMBDA_RETRY_ATTEMPTS: number = 3,
     public LAMBDA_STORAGE_SIZE: number = 10,
     public LAMBDA_TIMEOUT: number = 120,
-    public SNS_INPUT_TOPIC_NAME: string = "DIInputTopic",
-    public SNS_OUTPUT_TOPIC_NAME: string = "DIOutputTopic",
+    public SNS_INPUT_TOPIC_NAME: string = "osml-data-intake",
+    public SNS_STAC_TOPIC: string = "osml-stac-ingest",
```

Comment:
This somewhat breaks the naming pattern as other classes. I would prefer

- DIInputTopic or DIIntakeTopic
- DIStacTopic

Reply:
yeah I realized those naming patterns are actually non conventional and I was going to spin up a backlog to go back and update them

Reply:
One has `_NAME` and other doesn't. Could you pick one? Other than that, I would be a fan if we append `-topic` at the end of the string.

Reply:
Updated to have _NAME - not adding -topic to reduce redundant characters.